test/stdlib/distributions.jl

```diff
@@ -70,6 +84,7 @@ include(dir*"/test/test_utils/AllUtils.jl")
             # test is broken
             Weibull(2, 1),
             # Cauchy(0, 1),  # mean and variance are undefined for Cauchy
+            OrderedLogistic(0.5, [-1, 1])
```

Comment:
We don't need this any more. Please remove it from the list.

Reply:
done